microsoft-365/security/office-365-security/set-up-safe-links-policies.md

```diff
@@ -127,12 +127,12 @@ Creating a custom Safe Links policy in the Microsoft 365 Defender portal creates
 
 5. On the **URL & click protection settings** page that appears, configure the following settings:
 
-   - **Action on potentially malicious URLs within Emails** section:
+   - **Action on potentially malicious URLs within Emails(Email & Time of Click)** section:
      - **On: Safe Links checks a list of known, malicious links when users click links in email**: Select this option to turn on Safe Links protection for links in email messages. If you select this option, the following settings are available:
-       - **Apply Safe Links to email messages sent within the organization**: Select this option to apply the Safe Links policy to messages between internal senders and internal recipients.
-       - **Apply real-time URL scanning for suspicious links and links that point to files**: Select this option to turn on real-time scanning of links in email messages. If you select this option, the following setting is available:
-         - **Wait for URL scanning to complete before delivering the message**: Select this option to wait for real-time URL scanning to complete before delivering the message.
-       - **Do not rewrite URLs, do checks via SafeLinks API only**: Select this option to prevent URL wrapping. Safe Links is called exclusively via APIs at the time of URL click by Outlook clients that support it.
+       - **Apply Safe Links to email messages sent within the organization(Email – Intraorg & Time of Click)**: Select this option to apply the Safe Links policy to messages between internal senders and internal recipients. Turning this on will enable link wrapping for all intraorg messages.
+       - **Apply real-time URL scanning for suspicious links and links that point to files(Email)**: Select this option to turn on real-time scanning of links in email messages from external senders. If you select this option, the following setting is available:
+         - **Wait for URL scanning to complete before delivering the message(Email)**: Select this option to wait for real-time URL scanning to complete before delivering the message from senders external . Recommended setting is "On".
+       - **Do not rewrite URLs, do checks via SafeLinks API only(Time of Click)**: Select this option to prevent URL wrapping and skip reputation check during mailflow. Safe Links is called exclusively via APIs at the time of URL click by Outlook clients that support it.
```

Comment:
```suggestion
       - **Apply real-time URL scanning for suspicious links and links that point to files (Email)**: Select this option to turn on real-time scanning of links in email messages from external senders. If you select this option, the following setting is available:
         - **Wait for URL scanning to complete before delivering the message (Email)**: Select this option to wait for real-time URL scanning to complete before delivering the message from external senders. The recommended setting is **On**.
       - **Do not rewrite URLs, do checks via SafeLinks API only (Time of Click)**: Select this option to prevent URL wrapping and skip reputation check during mail flow. Safe Links is called exclusively via APIs at the time of URL click by Outlook clients that support it.
```